### ISSUE TITLE
Google Fonts module throws error when injected in nuxt-entry.js

### ISSUE THREAD

**Issue by Sananes** [nuxt-community/storybook#153]
### Version
@nuxtjs/storybook: v.3.0.0
nuxt: v2.14.7

### Reproduction Link
I don't unfortunately have a link as of yet

### Steps to reproduce
Add google fonts to your `nuxt.config.js`:

```
  buildModules: [
    [
      '@nuxtjs/google-fonts',
      {
        families: {
          Inter: {
            wght: [400, 500, 600, 700],
          },
          Roboto: true,
        },
        display: 'swap',
        prefetch: true,
      },
    ]
  ],
```

### What is Expected?
Storybook to not throw an error within the `.nuxt-storybook/storybook/nuxt-entry.js` and render correctly.

### What is actually happening?
An error keeps getting thrown when executing plugin script:

```
/**
 * @nuxtjs/storybook
 * execute plugin scripts
 */
window.__NUXT__ = { config: {} }
(function(){var l=document.createElement('link');l.rel="stylesheet";l.href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Roboto&display=swap";document.querySelector("head").appendChild(l);})();
```
Error in console:
```
nuxt-entry.js:14 Uncaught TypeError: {(intermediate value)} is not a function
    at Module.<anonymous> (nuxt-entry.js:14)
    at Module../.nuxt-storybook/storybook/nuxt-entry.js (nuxt-entry.js:224)
    at __webpack_require__ (bootstrap:853)
    at fn (bootstrap:150)
    at Module../.nuxt-storybook/storybook/entry.js (store.js:143)
    at __webpack_require__ (bootstrap:853)
    at fn (bootstrap:150)
    at Module../.storybook/storybook-init-framework-entry.js (preview.js-generated-config-entry.js:39)
    at __webpack_require__ (bootstrap:853)
    at fn (bootstrap:150)
```

### Insights
When I remove from buildModules google fonts it works out of the gate.

**Comment by farnabaz** [comment]
Hey @Sananes,
This should be fixed in #144, I will release the new version ASAP and ping you here 🙂

**Comment by Sananes** [comment]
> Hey @Sananes,
> This should be fixed in #144, I will release the new version ASAP and ping you here 🙂

Thank you very much my good man!

**Comment by farnabaz** [comment]
[v3.1.0](https://github.com/nuxt-community/storybook/releases/tag/v3.1.0) is out 🎉
